install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -5,7 +5,11 @@
 # -- namespaceOverride allows to override the destination namespace for Cilium resources.
 # This property allows to use Cilium as part of an Umbrella Chart with different targets.
 namespaceOverride: ""
-
+# @schema
+# type: [null, object]
+# @schema
+# -- commonLabels allows users to add a common label for all Cilium resources.
+commonLabels: {}
```

Comment:
I'm curious, as I understand we already apply `app.kubernetes.io/part-of: cilium` for various resources. Is the intent here that the user has their own specific labels they want Cilium to have?

Reply:
Yes, so they can apply specific OPA rules to cilium resources.